hedwiki/score/HED_score_0.0.2.mediawiki

```diff
@@ -219,62 +219,62 @@ This project was supported by the by the National Institute of Mental Health of
                 **** <nowiki># {takesValue}[free text]</nowiki>
-            *** Apnea-hypopnea-index-respiration-feature <nowiki>[events/h. Used with Findings-attributes tags]</nowiki>
+            *** Apnea-hypopnea-index-respiration <nowiki>[events/h. Used with Findings-attributes tags]</nowiki>
                 **** <nowiki># {takesValue}[free text]</nowiki>
             *** Periodic-respiration
                 **** <nowiki># {takesValue}[free text]</nowiki>
-            *** Tachypnea <nowiki>[cycles per min. Used with Findings-attributes tags]</nowiki>
+            *** Tachypnea-respiration <nowiki>[cycles per min. Used with Findings-attributes tags]</nowiki>
                 **** <nowiki># {takesValue}[free text]</nowiki>
-            *** Other-respiration-features
+            *** Other-respiration-feature
                 **** <nowiki># {takesValue}[free text]</nowiki>
     * ECG <nowiki>[electrocardiography]</nowiki>
         ** ECG-QT-period
             *** <nowiki># {takesValue}[free text]</nowiki>
         ** ECG-feature
-            *** ECG-Sinus-rhythm <nowiki>[Normal rhythm. Used with Findings-attributes tags]</nowiki>
+            *** ECG-sinus-rhythm <nowiki>[Normal rhythm. Used with Findings-attributes tags]</nowiki>
                 **** <nowiki># {takesValue}[free text]</nowiki>
-            *** ECG-Arrhythmia
+            *** ECG-arrhythmia
                 **** <nowiki># {takesValue}[free text]</nowiki>
-            *** ECG-Asystolia <nowiki>[duration: range in seconds. Used with Findings-attributes tags]</nowiki>
+            *** ECG-asystolia <nowiki>[duration: range in seconds. Used with Findings-attributes tags]</nowiki>
                 **** <nowiki># {takesValue}[free text]</nowiki>
-            *** ECG-Bradycardia <nowiki>[beats/min: range. Used with Findings-attributes tags]</nowiki>
+            *** ECG-bradycardia <nowiki>[beats/min: range. Used with Findings-attributes tags]</nowiki>
                 **** <nowiki># {takesValue}[free text]</nowiki>
-            *** ECG-Extrasystole
+            *** ECG-extrasystole
                 **** <nowiki># {takesValue}[free text]</nowiki>
-            *** ECG-Ventricular-premature-depolarization <nowiki>[Used with Findings-attributes tags]</nowiki>
+            *** ECG-ventricular-premature-depolarization <nowiki>[Used with Findings-attributes tags]</nowiki>
                 **** <nowiki># {takesValue}[free text]</nowiki>
-            *** ECG-Tachycardia <nowiki>[beats/min: range. Used with Findings-attributes tags]</nowiki>
+            *** ECG-tachycardia <nowiki>[beats/min: range. Used with Findings-attributes tags]</nowiki>
                 **** <nowiki># {takesValue}[free text]</nowiki>
-            *** Other-ECG-features
+            *** Other-ECG-feature
                 **** <nowiki># {takesValue}[free text]</nowiki>
     * EMG <nowiki>[electromyography]</nowiki>
-        ** EMG-Muscle-side
```

Comment:
Could we change `EMG` to `EMG-property` so that children of this node will satisfy "is-a".

Reply:
Question: The is-a seems (to me) to be correct now, but a more general question is whether we allow tags that require a child? In the case below, EMG-muscle-side will never be used without a child since it requires a specification of the muscle side. 
* EMG-channel-finding <nowiki> {suggestedTag=Finding-attribute/Significance} [electromyography]</nowiki>
        ** EMG-muscle-side
            *** EMG-left-muscle